ui/src/app/services/workspace-storage.service.ts

```diff
@@ -2,6 +2,7 @@ import 'rxjs/Rx';
 
 import {Injectable} from '@angular/core';
 
+
```

Comment:
Can the changes to this file be reverted? Agree per slack it probably makes sense to look at removing these storage services as a follow-up now that we have the workspace/user stores.